Packs/NCFTATheatIntel/Integrations/NCFTATheatIntel/NCFTATheatIntel.py

```diff
@@ -0,0 +1,205 @@
+            f"Allowed values: {', '.join(sorted(SHARING_GROUPS_MAP.keys()))}"
+        )
+
+    return ids
+
+
+def _safe_post(url: str, headers: Dict[str, str], payload: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    Execute POST to MISP and return parsed JSON or raise a useful error.
+    """
+    try:
+        resp = requests.post(url, headers=headers, json=payload, verify=not INSECURE, timeout=TIMEOUT)
+    except requests.exceptions.RequestException as e:
+        demisto.error(f"RequestException when calling MISP: {str(e)}")
+        raise DemistoException(f"Failed to call MISP: {str(e)}")
+
+    if resp is None:
+        raise DemistoException("No response received from MISP API.")
+
+    status = resp.status_code
+    text = resp.text or ""
+
+    if status != 200:
+        demisto.error(f"MISP returned status {status}: {text}")
+        raise DemistoException(f"MISP API returned status {status}: {text}")
+
+    try:
+        return resp.json()
+    except ValueError:
+        demisto.error(f"Failed to parse JSON from MISP response. Raw response: {text}")
+        raise DemistoException("MISP returned an invalid JSON response. See server logs for details.")
+
+
+# -----------------------
+# Command implementation
+# -----------------------
+def ncfta_get_iocs_command(args: Dict[str, Any]) -> CommandResults:
+    """
+    ncfta-get-iocs command:
+    - sharing_groups: comma-separated string of sharing group IDs (e.g. "4,6,7")
+    - timestamp: e.g. "7d", "24h", "6h" (default 7d)
+    - limit: limit number of IOC's in the response. Default: 100
+    """
+    sharing_groups_arg = args.get("sharing_groups", [])
+    timestamp = args.get("timestamp", "7d")
+    limit = int(args.get("limit", "100"))
+
+    sharing_group_ids = _parse_sharing_groups(sharing_groups_arg)
+
+    payload = {
+        "timestamp": timestamp,
+        "limit": limit,
+        "returnFormat": "json"
+    }
+
+    if sharing_group_ids:
+        payload["sharinggroup"] = sharing_group_ids
+
+    # Build request URL
+    if not BASE_URL:
+        raise DemistoException("Integration parameter 'url' is not configured.")
+    url = f"{BASE_URL}{ATTRIBUTES_ENDPOINT}"
+
+    # Perform the request
+    raw_response_json = _safe_post(url, HEADERS, payload)
+    response_json = raw_response_json.get("response", {}).get("Attribute", [])
+    response_json = _extract_minimal_attributes_to_display(response_json)
+    output_results = {
+        "count": len(response_json),
+        "results": response_json
+    }
+
+    return CommandResults(
+        outputs_prefix="NCFTA.Attributes",
+        outputs_key_field="",
+        outputs=output_results,
+        readable_output=f"Fetched {len(response_json)} attributes"
+    )
+
+
+""" MAIN FUNCTION """
+
+
+def main():
+    try:
+        command = demisto.command()
+        demisto.info(f"Command being called: {command}")
+        if command == "test-module":
+            return_results("ok")
```

Comment:
the test module command needs to test connectivity and not return ok straight away